frontend/src/containers/AuthContainer/Logout/Logout.js

```diff
@@ -0,0 +1,20 @@
+import React, { useEffect } from 'react';
+import { Redirect } from 'react-router-dom';
+import { connect } from 'react-redux';
+
+import { logout } from '../../../actions/index';
+
+const Logout = (props) => {
```

Comment:
@HiroFive Maybe it should be in component

Reply:
moved to components